dask-gateway.yaml

```diff
@@ -37,6 +40,21 @@ pipeline:
         -r dask-gateway/requirements.txt \
         --prefix=/usr --root=${{targets.destdir}}
 
+subpackages:
+  - name: dask-gateway-server
+    description: A multi-tenant server for securely deploying and managing Dask clusters
+    pipeline:
+      - name: Python Build
+        runs: |
+          cd dask-gateway-server
+          pip install -r requirements.txt --prefix=/usr --root=${{targets.subpkgdir}}
```

Comment:
I'm not sure how previous discussions went but does `pip install -r requirements.txt` mean we are installing python dependencies on the fly rather than wolfi packaged python dependencies?  If so, just checking we want to do this here?  I know we do this for maven rather than having to package up maven central but not sure if this is the same for python?

Reply:
fixed, thank you!